pointgrey_camera_driver/CMakeLists.txt

```diff
@@ -11,33 +11,55 @@ generate_dynamic_reconfigure_options(
   cfg/PointGrey.cfg
 )
 
-catkin_package(
-  CATKIN_DEPENDS roscpp nodelet
+catkin_package(CATKIN_DEPENDS
+  image_exposure_msgs nodelet roscpp sensor_msgs wfov_camera_msgs
 )
 
 add_definitions(-std=c++11)
 
 # If the user has manually installed flycapture, use the system path version. If not,
 # download it. We can't resolve this dependency using the usual rosdep means because
+
 # the Point Grey EULA prohibits redistributing the headers or the packages which
-# contains them. Fortunately, we can download the archive directly from their 
-# website during this build process.
-find_library(POINTGREY_LIB flycapture)
-if(NOT POINTGREY_LIB)
+# contains them. We work around this by downloading the archive directly from
```

Comment:
@cfoust can you help me out with this? I'm not exactly sure how all of your downloading stuff  works. Or lack there of